Pipfile

```diff
@@ -0,0 +1,142 @@
+chardet = "==3.0.4"
+click = "==6.7"
+cookies = "==2.2.1"
+coverage = "==4.4.2"
+cryptography = "==2.1.4"
+ddt = "==1.1.1"
+decorator = "==4.1.2"
+defusedxml = "==0.5.0"
+django-admin-rangefilter = "==0.3.1"
+django-choices = "==1.6.0"
+django-countries = "==5.0"
+django-debug-toolbar = "==1.9.1"
+django-environ = "==0.4.4"
+django-extensions = "==1.9.8"
+django-filter = "==1.1.0"
+django-money = "==0.12.1"
+django-redis = "==4.8.0"
+django-simple-email-confirmation = "==0.22"
+django-simple-history = "==1.9.0"
+django-taggit = "==0.22.1"
+django-vies = "==3.4.1"
+Django = "==1.11.8"
+djangorestframework-csv = "==2.0.0"
+djangorestframework = "==3.7.3"
+docker-compose = "==1.18.0"
+docker-pycreds = "==0.2.1"
+docker = "==2.7.0"
+dockerpty = "==0.4.1"
+docopt = "==0.6.2"
+docutils = "==0.14"
+dodgy = "==0.1.9"
+factory_boy = "==2.9.2"
+Faker = "==0.8.7"
+first = "==2.0.1"
+"flake8" = "==3.5.0"
+freezegun = "==0.3.8"
+google-api-python-client = "==1.6.4"
+gunicorn = "==19.7.1"
+honcho = "==1.0.1"
+"httplib2" = "==0.10.3"
+huey = "==1.5.5"
+idna = "==2.6"
+imagesize = "==0.7.1"
+ipdb = "==0.10.1"
+ipython_genutils = "==0.2.0"
+ipython = "==6.2.1"
+isort = "==4.2.15"
+jedi = "==0.11.1"
+"Jinja2" = "==2.10"
+jira = "==1.0.10"
+jsonschema = "==2.6.0"
+lazy-object-proxy = "==1.3.1"
+lxml = "==4.1.1"
+markupsafe = "==1.0"
+mccabe = "==0.6.1"
+"oauth2client" = "==4.1.2"
+oauthlib = "==2.0.6"
+olefile = "==0.44"
+parso = "==0.1.1"
+pbr = "==3.1.1"
+pdfkit = "==0.6.1"
+"pep257" = "==0.7.0"
+"pep8-naming" = "==0.4.1"
+pew = "==1.1.2"
+pexpect = "==4.3.1"
+pickleshare = "==0.7.4"
+pillow = "==4.3.0"
+pip-tools = "==1.11.0"
+pipenv = "==9.0.3"
+prompt-toolkit = "==1.0.15"
+prospector = "==0.12.7"
+"psycopg2" = "==2.7.3.2"
+ptyprocess = "==0.5.2"
+py-moneyed = "==0.7.0"
+"pyasn1-modules" = "==0.2.1"
+"pyasn1" = "==0.4.2"
+pycodestyle = "==2.0.0"
+pycparser = "==2.18"
+pydocstyle = "==2.1.1"
+pydrive = "==1.3.1"
+pyflakes = "==1.6.0"
+pygments = "==2.2.0"
+pylint-celery = "==0.3"
+pylint-common = "==0.2.5"
+pylint-django = "==0.7.2"
+pylint-flask = "==0.5"
+pylint-plugin-utils = "==0.2.6"
+pylint = "==1.7.4"
+pyopenssl = "==17.5.0"
+python-dateutil = "==2.6.1"
+pytz = "==2017.3"
+pyyaml = "==3.12"
+redis = "==2.10.6"
+refreshbooks = "==2.0"
+requests-oauthlib = "==0.8.0"
+requests-toolbelt = "==0.8.0"
+requests = "==2.18.4"
+requirements-detector = "==0.5.2"
+responses = "==0.8.1"
+retrying = "==1.3.3"
+rsa = "==3.4.2"
+setoptconf = "==0.2.0"
+simplegeneric = "==0.8.1"
+simplejson = "==3.13.2"
+snowballstemmer = "==1.2.1"
+sphinx = "==1.6.5"
+sphinxcontrib-websupport = "==1.0.1"
+sqlparse = "==0.2.4"
+suds-jurko = "==0.6"
+text-unidecode = "==1.1"
+texttable = "==0.9.1"
+traitlets = "==4.3.2"
+typing = "==3.6.2"
+unicodecsv = "==0.14.1"
+uritemplate = "==3.0.0"
+"urllib3" = "==1.22"
+virtualenv-clone = "==0.2.6"
+virtualenv = "==15.1.0"
+wcwidth = "==0.1.7"
+websocket-client = "==0.45.0"
+Werkzeug = "==0.13"
+wrapt = "==1.10.11"
+
+
+[dev-packages]
```

Comment:
@UmanShahzad Do we need to add this now if we're not specifying any packages below?

Reply:
@itsjeyd Since it was generated, and I didn't even know it existed, it might be good to leave around in case we decide to add stuff there (I can see that happening even now, but as a first time with Pipfile and considering things could change with their implementation, I'm not messing further with it except what was generated for me).